app/proxy/tests/test_metadata.py

```diff
@@ -170,106 +182,82 @@ class TestMetadataClientGetRemoteURLs(MetadataClientTest):
     def test_redis_get_error_raises_exception(self):
         self.mock_redis.setex.side_effect = redis.RedisError
 
-        embedly_data = self.get_mock_urls_data(self.sample_urls)
+        remote_data = self.get_mock_urls_data(self.sample_urls)
 
-        self.mock_requests_get.return_value = self.get_mock_response(
-            content=json.dumps(embedly_data))
+        self.metadata_client._make_remote_request = mock.Mock()
```

Comment:
We keep doing this everywhere just make that method always mocked then we don't need to make that custom subclass at all.